src/main/java/storagesystem/model/Organisation.java

```diff
@@ -0,0 +1,170 @@
+package storagesystem.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * An Organisation holds a collection of teams with the purpose being that the teams can communicate with each other.
+ * The teams that belong to an organisation should be relevant to one another.
+ * An Organisation should keep track of all the reservations between its teams.
+ */
+public class Organisation {
+    private final List<Team> teams = new ArrayList<>();
+    private final List<User> users = new ArrayList<>();
+    private String name;
+    //todo reservationHandler
+
+    public Organisation(String name) {
+        this.name = name;
+        //fill users from db
+        //fill teams from db
+    }
+
+    //deep copy
+    private Organisation(Organisation organisationToCopy) {
+        this.name = organisationToCopy.name;
+        this.teams.addAll(organisationToCopy.getTeams());
+        this.users.addAll(organisationToCopy.getUsers());
+        //todo reservationHandler = organisationToCopy.reservationHandlerDeepCopy
+    }
+
+    /**
+     * Exist to show a user all the items available to borrow.
+     *
+     * @return List of all the items from all the teams.
+     */
+    List<Item> getAllItems() {
+        List<Item> allItems = new ArrayList<Item>();
+        for (Team t :
+                teams) {
+            allItems.addAll(t.getAllItems());
+        }
+        return allItems;
+    }
+
+    /**
+     * The point of this method exist to show specific items for teams/users.
+     *
+     * @param ID every item has its own ID connected to it.
+     * @return the requested item if found
+     * @throws Exception if item ID not found
+     */
+    private Item getItem(int ID) throws Exception {
+        for (Team t :
+                teams) {
+            for (Item i :
+                    t.getAllItems()) {
+                //i.getID == ID
+                if (true) {
+                    System.out.println("Item found");
+                    return i;
+                }
+            }
+        }
+        throw new Exception("ItemID not found in list of items");
+    }
+
+    /**
+     * @param user
+     * @return
+     */
+    public List<Team> getUserTeams(User user) {
+        List<Team> userTeams = new ArrayList<Team>();
+        for (Team t : teams) {
+            for (int i : t.getAllMemberIDs()) {
+                if (user.getID() == i) {
```

Comment:
In Reservation and ReservationHandler i currently return copies of Reservations and not their ID's, should be consistent so need to change in one of the two places